### ISSUE TITLE
support for charged systems

### ISSUE THREAD

**Issue by bernstei** [SPARC-X/SPARC#29]
Are there any plans for SPARC to support charged systems (with a uniform background charge compensation)?

**Comment by phanish-suryanarayana** [comment]
Yes, we have already implemented it in some form. Will be incorporated into master very soon.
